internal/signer/windows/ncrypt/cert_util.go

```diff
@@ -1,7 +1,6 @@
 //go:build windows
 // +build windows
 
-// Cert_util provides helpers for working with Windows certificates via crypt32.dll
```

Comment:
Can you keep this comment as a file level comment? Maybe adding blank line between this and "package ncrypt" should get linter to pass? If that doesn't work, you can move it to after "package ncrypt"?